internal/rbd/encryption.go

```diff
@@ -111,6 +111,41 @@ func (ri *rbdImage) setupEncryption(ctx context.Context) error {
 	return nil
 }
 
+func (ri *rbdImage) copyEncryptionConfig(cp *rbdImage) error {
```

Comment:
good to have a source code comment about this newly introduced functiion @nixpanic

Reply:
Sure, I'll add that.